### PR TITLE
Feat vestings

### PR DESCRIPTION
**Pull request by MerlinEgalite**

### CODE REVIEW

File: create_vesting.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+if [[ -z ${1} ]]; then
+	echo "First argument (vester) not found, please set it and re-run the last command."
+	exit 1
+fi
+VESTER=$1
+
+if [[ -z ${2} ]]; then
+	echo "Second argument (signer) not found, please set it and re-run the last command."
+	exit 2
```
Comment (MathisGD) [inline create_vesting.sh]: 1 for general errors

```suggestion
	exit 1
```

File: create_vesting.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+echo "Cliff duration (eta) $CLIFF_DURATION"
+echo "Manager        (mgr) $MANAGER"
+echo "Restricted     (res) $RESTRICTED"
+echo "Blessed        (bls) $BLESSED"
+echo "Calldata:"
+
+echo "Call result:"
+
+
+cast call $VESTER $SIG $RECIPIENT $TOTAL $START $DURATION $CLIFF_DURATION $MANAGER $RESTRICTED $BLESSED --from SIGNER
```
Comment (MathisGD) [inline create_vesting.sh]:
```suggestion
cast call $VESTER $SIG $RECIPIENT $TOTAL $START $DURATION $CLIFF_DURATION $MANAGER $RESTRICTED $BLESSED --from $SIGNER
```

File: vestings_data.sh [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+if [[ -z ${1} ]]; then
+	echo "First argument (vester) not found, please set it and re-run the last command."
+	exit 1
+fi
+VESTER=$1
+
+if [[ -z ${2} ]]; then
+	echo "Second argument (signer) not found, please set it and re-run the last command."
+	exit 2
```
Comment (MathisGD) [inline vestings_data.sh:16]:
```suggestion
	exit 1
```

File: vestings_data.sh [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+while IFS=, read -r address amount; do
+
+	echo "New Vesting"
+
+	RECIPIENT=$address
+	TOTAL=$amount
+	SIMPLE_TOTAL=$(cast --to-unit "$amount" ether)
+	START=10000 # To write
+	DURATION=6 # To write
```
Comment (MathisGD) [inline vestings_data.sh]: Maybe it is cleaner to read it from the .env?

File: vestings_data.sh [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+	CLIFF_DURATION=0
+	MANAGER=$MANAGER_ADDRESS
+	RESTRICTED=true
+	BLESSED=true
```
Comment (MathisGD) [inline vestings_data.sh]: same here

**Review by MathisGD — APPROVED** [review]

**Review by adhusson — COMMENTED** [review]
lgtm

**Review by adhusson — APPROVED** [review]

**Pull request merged**